Meta/gn/secondary/Meta/Lagom/Tools/CodeGenerators/LibUnicode/BUILD.gn

```diff
@@ -1,16 +1,6 @@
 import("//Meta/Lagom/Tools/lagom_tool.gni")
 
-lagom_tool("GenerateUnicodeData") {
-  sources = [ "GenerateUnicodeData.cpp" ]
-  deps = [ "//Userland/Libraries/LibMain" ]
-}
-
 lagom_tool("GenerateEmojiData") {
```

Comment:
presumably we can remove the entire directory here, the EmojiData generator is also unused on master.

So, we can remove Meta/Lagom/Tools/CodeGenerators/LibGL, LibLocale, LibTimeZone and LibUnicode.

Nice catch, I forgot to look at the code generator rules.

Reply:
Yeah, we can remove those other tools as well, I had those locally removed but not merged into the commit 🤦